Guides/NUI/Components: Added namespace info and deprected note (#1274)
Samsung/tizen-docs: docs/application/dotnet/guides/nui/nui-components/Slider.md

![Slider](./media/slider.png) ![Slider](./media/slider2.png)

> ## Add namespace using
> Slider is defined in namespace Tizen.NUI.Components.

Reviewer: Please verify the following suggestion for technical accuracy.

```suggestion
## Add namespace
To implement slider, add namespace `using Tizen.NUI.Components` in your application:
```
